### PR TITLE
README: extension customization

### PR DESCRIPTION
**Pull request by bryanlatten**
Added additional instructions to enable and disable extension

### CODE REVIEW

**Review by bossjones — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -36,11 +36,13 @@ Adding startup logic, [basic](https://github.com/behance/docker-base#startuprunt
...
-- Extra PHP Modules:
+- Common PHP extensions:
+
+For extension customization, including enabling and disabling defaults, see [here](https://github.com/bryanlatten/docker-php#downstream-configuration)
 
 `*`  - not available on `-alpine` variant  
-`^`  - not available on `7.2` 
-`~`  - disabled by default (default: use `phpenmod` to enable, Alpine-only: uncomment .ini file)
+`^`  - not available on `7.2`  
+`~`  - disabled by default
```
Comment (mannytoledo) [inline README.md:45]: Those markdown newlines are invaluable.

Comment (bryanlatten) [reply README.md:45]: Yes, *very* valuable. the linebreak doesn't take place unless there are two of them, there is currently one.

Comment (mannytoledo) [reply README.md:45]: I think that's what I just said? I actually had to google it because I doubted myself, ha.

image

Comment (bryanlatten) [reply README.md:45]: haha, my apologies. I read it too fast.

**Pull request merged**